osu.Game/Screens/Select/FooterV2/FooterButtonModsV2.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+
+namespace osu.Game.Screens.Select.FooterV2
+{
+    public partial class FooterButtonModsV2 : FooterButtonV2
+    {
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colour)
+        {
+            Text = "Mods";
+            Icon = FontAwesome.Solid.ArrowsAlt;
```

Comment:
If we are still following the figma designs, this should be:
```suggestion
            Icon = FontAwesome.Solid.ExchangeAlt;
```

Reply:
my bad, checked the wrong commit on the old branch